selfdrive/car/toyota/interface.py

```diff
@@ -268,7 +268,8 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
 
     if ret.enableGasInterceptor:
       set_long_tune(ret.longitudinalTuning, LongTunes.PEDAL)
-    elif candidate in [CAR.COROLLA_TSS2, CAR.COROLLAH_TSS2, CAR.RAV4_TSS2, CAR.RAV4H_TSS2, CAR.LEXUS_NX_TSS2]:
+    elif candidate in [CAR.COROLLA_TSS2, CAR.COROLLAH_TSS2, CAR.RAV4_TSS2, CAR.RAV4H_TSS2, CAR.LEXUS_NX_TSS2,
+                       CAR.HIGHLANDER_TSS2, CAR.HIGHLANDERH_TSS2]:
```

Comment:
perhaps blacklist the other cars so the default is TSS2

Reply:
Yeah wanted to do that in another PR